plugins/Goals/templates/_formAddGoal.twig

```diff
@@ -120,6 +120,13 @@
             </div>
         </div>
 
+        <div piwik-field uicontrol="checkbox" name="use_event_value"
```

Comment:
Can we move this after case sensitive? That's because case sensitive is much more valuable option and more often used maybe than the event option and be good for consistency to always have case sensitive directly below. All good though to keep it also.

Reply:
be good to add some inline help to explain what it does actually

Reply:
Was hoping the translated label would be clear enough, is there something confusing about it or something that needs explanation?

Reply:
![image](https://user-images.githubusercontent.com/273120/46108619-9d6b2c00-c232-11e8-9762-1d9ffbf2e359.png)

It would be good to say something like:

"Enable this setting if you are sending an event value along with your event tracking request. Any number you send as event value will be also used as the revenue instead of the configured Goal revenue. It may be useful to enable this option if the event value you are tracking also matches the revenue it is worth to you. If the revenue is always the same and does not vary per conversion, we recommend configuring the goal default revenue below."

There might be more we want to add. I'm sure @mattab has some thoughts on this. I think there are only very few cases when you want to enable this setting as it can mess with your revenue quite a bit.

Can we maybe also disable the setting "Goal default revenue is (optional)" setting when this setting is enabled? Or maybe we show the setting there? By the looks in the code it would always overwrite the revenue with the event revenue and never use the goal revenue.

Reply:
could work. would need to see it though I guess. unless @mattab has any specific thoughts

Reply:
Sounds good to put them together and add some inline help as suggested. the label is clear but not so much if you are not familiar with the concepts, so the descriptive inline help text makes things more clear and removes any possible confusion👍